packages/angular/src/migrations/update-13-3-0/update-mfe-webpack-config.ts

```diff
@@ -0,0 +1,128 @@
+import { Tree } from '@nrwl/devkit';
+import { logger, formatFiles } from '@nrwl/devkit';
+import { tsquery } from '@phenomnomnominal/tsquery';
+import { forEachExecutorOptions } from '@nrwl/workspace/src/utilities/executor-options-utils';
+import { Identifier, Node, StringLiteral } from 'typescript';
+
+export default async function (tree: Tree) {
+  const NRWL_WEBPACK_BROWSER_BUILDER = '@nrwl/angular:webpack-browser';
+  const CUSTOM_WEBPACK_OPTION = 'customWebpackConfig';
+
+  forEachExecutorOptions(
+    tree,
+    NRWL_WEBPACK_BROWSER_BUILDER,
+    (opts, projectName) => {
+      // Update the webpack config
+      const webpackPath = opts[CUSTOM_WEBPACK_OPTION].path;
+      if (!tree.exists(webpackPath)) {
+        logger.warn(
+          `Webpack config file for project: ${projectName} does not exist. Skipping project.`
+        );
+        return;
+      }
+      const webpackConfig = tree.read(webpackPath, 'utf-8');
+      if (!webpackConfig.includes('ModuleFederationPlugin')) {
+        logger.warn(
+          `Webpack config file for project: ${projectName} is not using Module Federation. Skipping project.`
```

Comment:
I wouldn't output a warning. There's nothing wrong with not using Module Federation and a warning might cause noise and some confusion. If a project doesn't have it, it means the migration shouldn't process the project and imho that should happen silently.

Wdyt?

Reply:
Ah I was thinking that executor would only be used with MFEs but I guess it doesn't necessarily have to be

Reply:
Yea, I wouldn't assume that is only used for MFEs, custom builders to add custom webpack config exists way before MFE, and devs use them for several reasons.